packages/ui/__stories__/RadioGroup.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useRef, useState } from 'react'
+import { v4 as uuid } from 'uuid'
+import { Radio } from '../src/Radio'
+import { logger } from '@hazelcast/services'
+import { RadioGroup } from '../src/RadioGroup'
+import { Form, Formik } from 'formik'
+import { RadioGroupFieldFormik } from '../src/RadioGroupFormik'
+import { RadioFieldFormik } from '../src/RadioFormik'
+
```

Comment:
Can we also have an `inline` radio group story?

Reply:
@danad02 we do have it, just in a radio story. They're quite related and the only reason they're separated is so that we have a documentation for both of them available. I can move the inline scenario to RadioGroup stories though.

Reply:
@jvorcak Moving it here would make sense, thanks :+1: